bin/restart

```diff
@@ -11,4 +11,4 @@ docker network ls | grep `basename $(pwd)` | awk '{ print $1 }' | xargs docker n
 # docker images delete all 3
 docker images | grep `basename $(pwd)` | awk '{ print $1 }' | xargs docker image rm -f
 
-docker compose up -d
\ No newline at end of file
+docker-compose up -d
```

Comment:
Hmm... I'm not sure if this is the right answer. I think `docker compose` and `docker-compose` are *both* legitimate commands? should we do something like 

```bash
COMPOSE=docker-compose
if [ -x $COMPOSE ]; then
    echo "$COMPOSE exists."
else
    COMPOSE="docker compose"
    if [ -x $COMPOSE ]; then
        echo "$COMPOSE exists."
    else
        echo "Cannot find docker compose. Please make sure you have your shit set up right."
        exit 1
    fi
fi
```